engine/main.py

```diff
@@ -628,9 +629,12 @@ def unsubscribe_group(group_name, user):
     res = {'status':False}
     try:
         group = Group.objects.get(name=group_name)
-        membergroup = MemberGroup.objects.get(group=group, member=user)
-        membergroup.delete()
-        res['status'] = True
+        membergroups = MemberGroup.objects.filter(group=group)
+        membergroup = membergroups.get(member=user)
```

Comment:
This line can be shortened to 1 line. You can search MemberGroup for both group and member at the same time